### PR TITLE
add blog post for release 1.4

### LINKED ISSUE #4902: Knative 1.4 release notes blog post
## Tasks

After Knative 1.4 release day (April 19, 2022):
1. After each component has released, consolidate all release notes into a blog post similar to the [Knative 1.2 blog post](https://knative.dev/blog/releases/announcing-knative-v1-2-release/). Use the [template](https://github.com/knative/docs/blob/main/blog/docs/releases/_template-announcing-knative-v1-X-release.txt) to format the blog post correctly.
2. Add the new blog page to `blog/config/nav.yml`
3. Update the "featured posts" section of `blog/docs/index.md` with the new release. 

## Resources

Add the following release notes to the post:
- [ ] https://github.com/knative/serving/releases/tag/knative-v1.4.0
- [ ] https://github.com/knative/eventing/releases/tag/knative-v1.4.0
- [ ] https://github.com/knative-sandbox/eventing-kafka-broker/releases/tag/knative-v1.4.0
- [ ] https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.4.0
- [ ] https://github.com/knative/client/blob/main/CHANGELOG.adoc#v140-2022-04-19
- [ ] https://github.com/knative/operator/releases/tag/knative-v1.4.0

### PR DESCRIPTION
**Pull request by nader-ziada**
Fixes #4902

## Proposed Changes 

- blog post for release 1.4

### CODE REVIEW

**Comment by nader-ziada** [comment]
@rhuss I added the client release notes section here from the changelog, please take a look

File: blog/docs/releases/announcing-knative-v1-4-release.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.4.0 -->
+
+
+#### 💫 New Features & Changes
+
+- Now performance graphs for the RabbitMQ Broker are going to be generated for every mayor version release inside our performance test directories (#668, @gabo1208)
+- Now the RabbitMQ Source, when failing to send an event, does not get stuck in a cycle requeueing messages as they come. It follows a retries based backoff strategy (and does not reqeue them if the prefetchCount is 1) (#614, @gabo1208)
+- Now the RabbitMQ's Broker Ingress pod can recover from a closed Channel or Connection (#648, @gabo1208)
+- 🧹 Changed prefetchCount env variable name to Parallelism in both Trigger and Source, to better reflect its functionality (#676, @gabo1208)
+- Remove support for managed RabbitMQ brokers (#708, @gab-satchi)
```
Comment (xtreme-sameer-vohra) [inline blog/docs/releases/announcing-knative-v1-4-release.md:122]: Can this be under a `🚨 Breaking or Notable Changes` section as its removing existing functionality?
```suggestion


#### 🚨 Breaking or Notable Changes

- Remove support for managed RabbitMQ brokers (#708, @gab-satchi)
```

File: blog/docs/releases/announcing-knative-v1-4-release.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/knative-v1.4.0 -->
+
+### 🚨 Breaking or Notable Changes
+
+- The minimum Kubernetes version is 1.22. (#6280, @pierDipi)
+
+### 💫 New Features & Changes
+
+- The sugar reconciler has been integrated into the base eventing controller and is now controlled by two LabelSelector fields.
```
Comment (xtreme-sameer-vohra) [inline blog/docs/releases/announcing-knative-v1-4-release.md:64]: Can this be under a `🚨 Breaking or Notable Changes` section as its modifying existing functionality?

**Review by xtreme-sameer-vohra — CHANGES_REQUESTED** [review]
Thanks Nader
Requested 2 changes based on context I happened to have on the features.

File: blog/docs/releases/announcing-knative-v1-4-release.md (hunk 2 above)
Comment (nader-ziada) [reply blog/docs/releases/announcing-knative-v1-4-release.md:61]: done

File: blog/docs/releases/announcing-knative-v1-4-release.md (hunk 1 above)
Comment (nader-ziada) [reply blog/docs/releases/announcing-knative-v1-4-release.md]: done

File: blog/docs/releases/announcing-knative-v1-4-release.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+- Refactor the post-install yaml files for API migration (#996, thanks @houshengbo)
+- Fix the RBAC issue for Knative due to the label change (#1010, thanks @houshengbo)
+- Set capability to Seamless Upgrades (#1014, thanks @pierDipi)
+- Implement the label and annotation overwriting for service (#1033, thanks @houshengbo)
+- Adding code of conduct to reference knative/community's CoC (#1003, thanks @rhuss)
+
+
+### 🐞 Bug Fixes
+
+- Change int32 into *int32 for Replicas (#1029, thanks @houshengbo)
```
[the remark this answers is not in the document]
Comment (nader-ziada) [reply blog/docs/releases/announcing-knative-v1-4-release.md]: done

File: blog/docs/releases/announcing-knative-v1-4-release.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+## Announcing Knative 1.4 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in
+[Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+## Table of Contents
+- [Highlights](#highlights)
+- [Serving v1.4](#serving-v1.4-NOSPACES)
```
Comment (csantanapr) [inline blog/docs/releases/announcing-knative-v1-4-release.md:21]: These links are broken, something to do with `NOSPACES`?
Check out the 1.3 release markdown https://raw.githubusercontent.com/knative/docs/main/blog/docs/releases/announcing-knative-v1-3-release.md

File: blog/docs/releases/announcing-knative-v1-4-release.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+- Our minimum supported Kubernetes version is now 1.22
+
+
+## Serving v1.4
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/knative-v1.4.0 -->
+
+🚨 Breaking or Notable
+ - Our minimum supported Kubernetes version is now 1.22 (#12753, @dprotaso)
```
Comment (csantanapr) [inline blog/docs/releases/announcing-knative-v1-4-release.md:41]: All entries like `(#12753, @dprotaso)` need to only have the PR number, remove the user id it should be moved to the end of the document. something like `([#12753](https://github.com/knative/serving/pull/12753))`

**Review by csantanapr — CHANGES_REQUESTED** [review]
sorry that I didn't the broken links on the first passed.

File: blog/docs/releases/announcing-knative-v1-4-release.md (hunk 4 above)
Comment (nader-ziada) [reply blog/docs/releases/announcing-knative-v1-4-release.md]: fixed links

File: blog/docs/releases/announcing-knative-v1-4-release.md (hunk 5 above)
Comment (nader-ziada) [reply blog/docs/releases/announcing-knative-v1-4-release.md]: updated all RPs links

**Review by csantanapr — APPROVED** [review]
great thanks @nader-ziada 🎉

**Pull request merged**